### ISSUE TITLE
How to increase accuracy ?

### ISSUE THREAD

**Issue by baoloc97** [OpenNMT/OpenNMT-py#2076]
I use this model for JA-EN. But when i go 9500 step it has accuracy ~ 61% then it stuck here. Cant go higher even at 13000 step accuracy only 60. So how can i improve this. I have 3.4 million rows for train. Here is my config
* tgt vocab size: 21348.
* src vocab size: 18582.
* save_model: toy-ende/run/model_4milsv4_JaEn
save_checkpoint_steps: 5000
train_steps: 15000
valid_steps: 10000

src_word_vec_size : 4000
tgt_word_vec_size : 4000
batch_size : 80
learning_rate : 0.001
rnn_size : 512
optim : 'adam'
layers : 2
dropout : 0.3
log_file: toy-ende/run/logsv4.txt

And one more thing there are a lot of unknow word when i use translate so how to fix it.

**Comment by francoishernandez** [comment]
There is no magic recipe, you'll have to experiment.
I encourage you to have a look around the [forum](https://forum.opennmt.net) and search for topic such as "subwords" tokenization, backtranslation, etc.
You can alsow switch to the transformer architecture.
